cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+    "video": false
```

Comment:
When tests fail, failure artifacts are added to cypress/screenshots and cypress/videos directories. This line disables capturing videos.